src/main/kotlin/no/skatteetaten/aurora/boober/feature/AlertsFeature.kt

```diff
@@ -173,29 +180,67 @@ class AlertsFeature : Feature {
     private fun AuroraDeploymentSpec.extractConfiguration(alertName: String): AlertConfiguration {
         val confPath = "$featureName/$alertName"
 
-        val alertEnabled = this.getOrNull<Boolean>("$confPath/enabled")
-        val alertExpr = this.getOrNull<String>("$confPath/expr")
-            ?: throw IllegalStateException("Missing $confPath/expr value, check validation-logic")
-        val alertDelay = this.getOrNull<String>("$confPath/delay")
-        val alertConnection = this.getOrNull<String>("$confPath/connection")
-        val alertSeverity = this.getOrNull<String>("$confPath/severity")
-            ?: throw IllegalStateException("Missing $confPath/severity value, check validation-logic")
-        val alertSummary = this.getOrNull<String>("$confPath/summary") ?: "oppsummering av alarm er ikke angitt"
-        val alertDescription = this.getOrNull<String>("$confPath/description") ?: "beskrivelse av alarm er ikke angitt"
+        // non-nullable w. no-default
+        val confExpression = this.get<String>("$confPath/expr")
+        val confSeverity = this.get<String>("$confPath/severity")
+
+        // nullable w. conf-default + fallback
+        val confEnabled = this.getWithDefaultFallback(confPath, "enabled", false, defaultsName)
```

Comment:
Vi har en metode som heter this.getOrDefault(), er det mulig å bruke den istedenfor? 

Se kommentaren nedenfor ang. confSummary for å løse det behovet for denne metoden

Reply:
getOrDefault() krever at propertien er satt på alarmen eller i alertDefaults, alternativt kan det implementeres en ekstra getOrDefault som kan returnere null

Reply:
Hva hvis du legger på default verdier for defaults feltene:
            AuroraConfigFieldHandler("$defaultsName/enabled", defaultValue = false),
            AuroraConfigFieldHandler("$defaultsName/connection"),
            AuroraConfigFieldHandler("$defaultsName/connections"),
            AuroraConfigFieldHandler("$defaultsName/delay", defaultValue = 1)

Reply:
gikk over validering igjen og så at getOrDefault vil fungere fint